<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AddWindow</class>
 <widget class="QMainWindow" name="AddWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>371</width>
    <height>295</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>10</y>
      <width>311</width>
      <height>181</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="number_label">
       <property name="text">
        <string>学号：</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="number_edit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="name_label">
       <property name="text">
        <string>姓名：</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="name_edit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="grade_label">
       <property name="text">
        <string>年级：</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="grade_edit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="acadamy_label">
       <property name="text">
        <string>学院：</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="acadamy_edit"/>
     </item>
     <item row="4" column="0">
      <widget class="QLabel" name="majory_label">
       <property name="text">
        <string>专业：</string>
       </property>
      </widget>
     </item>
     <item row="4" column="1">
      <widget class="QLineEdit" name="majory_edit"/>
     </item>
     <item row="5" column="0">
      <widget class="QLabel" name="phone_label">
       <property name="text">
        <string>电话：</string>
       </property>
      </widget>
     </item>
     <item row="5" column="1">
      <widget class="QLineEdit" name="phone_edit"/>
     </item>
    </layout>
   </widget>
   <widget class="QPushButton" name="sure_button">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>220</y>
      <width>80</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>添加</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>371</width>
     <height>17</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
